Parse/cloud/main.js

```diff
@@ -355,6 +364,59 @@ Parse.Cloud.define("postVent", async (request) => {
   return;
 });
 
+Parse.Cloud.define("createGroup", async (request) => {
+  const Group = Parse.Object.extend("GroupDetails");
+  const group = new Group();
+  const authorId = request.params.authorId;
+  var groupMembershipIds = request.params.groupMembershipIds;
+  var groupName = request.params.groupName;
+
+  const userQuery = new Parse.Query("User");
+  userQuery.equalTo("objectId", authorId);
+  const user = await userQuery.first();
+
+  if (user == undefined) {
+    return;
+  }
+
+  group.set("groupAuthor", user);
+  group.set("groupName", groupName);
+
+  group.save()
+  .then((group) => {
+  }, (error) => {
+    alert('Failed to create new object, with error code: ' + error.message);
+    return;
+  });
+
+  var groupMembershipArray = [];
+  for (var i = 0; i < groupMembershipIds.length; i++) {
+    let currentMembershipId = groupMembershipIds[i];
+
+    const GroupMembership = Parse.Object.extend("GroupMembership");
+    const groupMembership = new GroupMembership();
+
+    const userQuery = new Parse.Query("User");
+    userQuery.equalTo("objectId", currentMembershipId);
+    const user = await userQuery.first();
+
+    groupMembership.set("group", group);
+
+    if (user !== undefined) {
+      groupMembership.set("user", user);
+    }
+    else {
+      console.log("not a user");
```

Comment:
I believe javascript supports `console.error("not a user")` for errors but I could be wrong.